src/core/utils/useLoans.tsx

```diff
@@ -0,0 +1,138 @@
+import { useGetLoansV2 } from "../fbs/fbs";
+import { useGetV1UserLoans } from "../publizon/publizon";
+import { useConfig } from "./config";
+import {
+  filterLoansNotOverdue,
+  filterLoansOverdue,
+  filterLoansSoonOverdue,
+  getDueDatesLoan,
+  sortByDueDate
+} from "./helpers/general";
+import {
+  mapFBSLoanToLoanType,
+  mapPublizonLoanToLoanType
+} from "./helpers/list-mapper";
+import { LoanType } from "./types/loan-type";
+import { ThresholdType } from "./types/threshold-type";
+
+type Loans = {
+  overdue: LoanType[];
+  soonOverdue: LoanType[];
+  farFromOverdue: LoanType[];
+  isLoading: boolean;
+  isError: boolean;
+  sortedByDate?: LoanType[];
+  stackedMaterialsDueDates?: string[];
+};
+
+type UseLoansType = {
+  all: Loans & {
+    loans: LoanType[];
```

Comment:
Consider moving `loans` to the `Loans` type. 

I wonder why we do not think it is relevant to return all loans from either `fbs` or `publizon` going forward.